### ISSUE TITLE
Triggered by muscle relaxant

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7412]
**Preferred term label:**
Triggered by muscle relaxant

**Synonyms**

**Definition (free text, please give PubMed ID)**
Applies to a sign or symptom that is provoked or brought about by exposure to a muscle relaxant such as suxamethonium and mivacurium.

**Parent term (use hpo.jax.org/app)**
Triggered by 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:617936 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
